benchmarks/Package.swift

```diff
@@ -0,0 +1,59 @@
+// swift-tools-version: 5.7
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCertificates open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftCertificates project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftCertificates project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import PackageDescription
+
+let package = Package(
+    name: "benchmarks",
+    platforms: [
+        .macOS(.v13),
+    ],
+    dependencies: [
+        .package(path: "../"),
+        .package(url: "https://github.com/ordo-one/package-benchmark", from: "1.9.0"),
+        .package(url: "https://github.com/apple/swift-crypto.git", from: "2.5.0"),
+        .package(url: "https://github.com/apple/swift-asn1.git", .upToNextMinor(from: "0.10.0")),
+    ],
+    targets: [
+        .executableTarget(
+            name: "CertificatesBenchmarks",
+            dependencies: [
+                "Sources",
+                .product(name: "Benchmark", package: "package-benchmark"),
+            ],
+            path: "Benchmarks/CertificatesBenchmarks",
+            plugins: [
+                .plugin(name: "BenchmarkPlugin", package: "package-benchmark")
+            ]
+        ),
+        .target(
+            name: "Sources",
+            dependencies: [
+                .product(name: "Benchmark", package: "package-benchmark"),
+                .product(name: "X509", package: "swift-certificates"),
+                .product(name: "SwiftASN1", package: "swift-asn1"),
+                .product(name: "Crypto", package: "swift-crypto"),
+            ],
+            path: "Sources"
+        ),
+        .testTarget(
+            name: "Tests",
+            dependencies: [
+                "Sources",
+            ],
+            path: "Tests"
+        )
```

Comment:
I find the naming and paths here a bit weird. Can't we come up with something in the default path format?

Reply:
Changed